src/app/(auth)/(navigationsBarLayout)/mypage/ui/Heatmap.tsx

```
@@ -37,10 +36,39 @@ export const Heatmap = ({ data }: { data: IHeatmapItem[] }) => {
   }, [tab, data]);
 
   return (
-    <div className="flex flex-row gap-10 flex-1">
+    <div
+      className="flex flex-row gap-10 flex-1 relative"
+      onMouseLeave={() => {
+        setHoverData(null);
+        setTooltipPos(null);
+      }}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)